design/ceph/ceph-nfs-ganesha.md

```diff
@@ -48,18 +48,39 @@ This allows the NFS-Ganesha server cluster to be scalable and highly available.
 
 ### Ceph NFS-Ganesha CRD
 
-The NFS-Ganesha CRD will specify the following:
-
+#### Server settings
 - Number of active Ganesha servers in the cluster
-
 - Placement of the Ganesha servers
-
 - Resource limits (memory, CPU) of the Ganesha server pods
 
-- RADOS pool and namespace where backend objects will be stored (supplemental
-  config objects and recovery backend objects)
-
-
+#### Security settings
+
+- Using the System Security Services Daemon ([SSSD])
+  - This can be used to connect to many ID management services, but LDAP is tested and supported
+  - The first option implemented is to run SSSD in a sidecar with the intent to add an option in the
```

Comment:
How about stating the design plan for the first implementation and break out the future work into a separate bullet?
```suggestion
  - Run SSSD in a sidecar
```